### ISSUE TITLE
[Bug]: no billing reported using rerank endpoint and amazon bedrock 

### ISSUE THREAD

**Issue by ladrians** [BerriAI/litellm#7258]
### What happened?

I am testing the liteLLM `rerank` endpoint against bedrock; for `us-west-2` region I got access to `cohere.rerank-v3-5:0` and `amazon.rerank-v1:0`. Noticed that no billing is reported; both models the meta section details

### Relevant log output

```shell
"meta": {
        "billed_units": {},
        "tokens": {}
    }
```

### Are you a ML Ops Team?

No

### What LiteLLM version are you on ?

main-v1.55.2

**Comment by ghost** [comment]
I don't believe bedrock returns these values. I think they charge just on number of requests. 

Let me know if I'm mistaken @ladrians

**Comment by hollamad** [comment]
They charge based on requests and the response doesn't include usage info. 
Ref: example response [here](https://opensearch.org/docs/latest/ml-commons-plugin/tutorials/reranking-bedrock/), and from the [pricing page](https://aws.amazon.com/bedrock/pricing/)  

![Image](https://github.com/user-attachments/assets/746d25f2-11d1-4c4c-ad90-145460128a47)

**Comment by ladrians** [comment]
got it, I though it would be nice to still generate a usage based on the usage and to keep the "unified response".
Feel free to close it, thanks!

**Comment by ghost** [comment]
What value would you expect there? @ladrians

**Comment by ladrians** [comment]
to fill-in the billing with something like this?

```
"billed_units": {
      "search_units": 1
  }
```

**Comment by ilarischeinin** [comment]
It would be nice to have the costs tracked, but I understand that is not trivial when the response doesn't include any usage stats. Both the Amazon and Cohere rerank models are priced per request, but if there are more than 100 documents, it'll count as multiple requests. And documents longer than 512 tokens will be broken down into multiple documents.

Amazon:
![](https://cdn.zappy.app/7ee45dc8f5239ec87f361d664bd6d7d7.png)

Cohere:
![](https://cdn.zappy.app/3352f30bf11c6eba1496b53f9a72435a.png)

**Comment by ghost** [comment]
Adding this to our feb 2025 roadmap - https://github.com/BerriAI/litellm/discussions/8375

cc: @vibhavbhat as an fyi

**Comment by ghost** [comment]
picking this up

**Comment by ghost** [comment]
Implemented the v0 for cost tracking on bedrock

plan for tomorrow is to handle the query calculation @ilarischeinin shared
